### ISSUE TITLE
[Bug]: jest_test fails with usage statement on windows

### ISSUE THREAD

**Issue by peakschris** [aspect-build/rules_jest#322]
### What happened?

On Windows, jest_test fails with a usage message related to the --config parameter. It turns out it is because rules_jest is creating this parameter:

`--config 'myconfig.js'`

However the correct parameter on Windows is:

`--config "myconfig.js"`

### Version

Development (host) and target OS/architectures:
Windows

Output of `bazel --version`:
7.6.1

Version of the Aspect rules, or other relevant rules from your
`WORKSPACE` or `MODULE.bazel` file:
rules_js 2.4.2
rules_jest 0.23.3

Language(s) and/or frameworks involved:

### How to reproduce

```shell

```
